.github/workflows/push.yml

```diff
@@ -9,8 +9,11 @@ jobs:
   ci:
     name: CI
     uses: ./.github/workflows/ci.yml
+    secrets: inherit
```

Comment:
Secrets don't get passed by default. This was why our build times have balloned.